crates/fluvio-stream-dispatcher/src/dispatcher/k8_dispatcher.rs

```diff
@@ -38,58 +34,46 @@ static SC_RECONCILIATION_INTERVAL_SEC: Lazy<u64> = Lazy::new(|| {
 });
 
 /// For each spec, process updates from Kubernetes metadata
-pub struct K8ClusterStateDispatcher<S, C>
+pub struct ClusterStateDispatcher<S, C, M>
```

Comment:
I agree that 'ClusterStateDispatcher' might not be relevant anymore, but `MetadataStore` is also not precise, since it is not a store per se, it is still closer to the dispatcher than to the store, therefore I renamed it to `MetadataDispatcher`.

Reply:
I would not rename the crate in this PR since it is published and too much effort right now.

Reply:
fair enough